README.md

```diff
@@ -167,14 +160,14 @@ disease:
 
 then the `c01` would be mapped to both `output_class_c` and `disease`. This scheme allows many-to-many mappings.
 
-Records assigned to classes that are not mapped will be dropped, therefore will not exist in the output dataset. Default mapping behavior, that occurs when class mapper is unspecified, maps classes one to one, ex: `c01 -> c01` and assumes that `h01-h07` and `b02` are healthy classes.
+Records assigned to class that are not mapped will be dropped, therefore will not exist in the output dataset. Default mapping behavior, that occurs when class mapper is unspecified, maps classes one to one, ex: `c01 -> c01` and assumes that `h01-h07` and `b02` are healthy classes.
 
-##### Healthy flag
-In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as healthy. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
-- if the mask belongs to a healthy class **and** segmentation is not binary, the color is white.
-- otherwise, the color is black.
+##### Positive flag
+In ERS dataset, some masks are empty files, especially the ones that label healthy classes. By default, such mask files are ignored unless they belong to a class labeled as positive. The script converts them to a valid picture which mode and size is based on the original frame. Filling color of the picture is determined by the following rule:
```

Comment:
This positive flag now has nothing to do with the healthy or not healthy. It works only for binary segmentation and works as following:
1. If a class is postiive one, then masks that belong to it are merged 
2. If class is not positive, then the mask is going to be fully black

Reply:
Done